_data/simple-icons.json

```diff
@@ -4417,6 +4417,14 @@
             "hex": "4D148C",
             "source": "https://newsroom.fedex.com"
         },
+        {
+            "title": "Fediverse",
+            "hex": "111111",
```

Comment:
I am not sure about this HEX. Can @simple-icons/icon-review chip in and suggest a better color if any?

Reply:
Possibly #FFFFFF could work?

Reply:
We're trying to avoid `#FFFFFF` as a hex on icons, as it's rarely the case. In this case, I'd go by whatever colour is used most frequently when referring to the Fediverse. Can't find _one_ solid choice though.

Reply:
I've honestly seen it represented as a ton of different colors, but yellow has been used a lot.